app/app/settings.py

```diff
@@ -45,7 +45,7 @@
     'django.contrib.staticfiles',
     'django_htmx',
     'django_browser_reload',
-    'public',
+    'django.contrib.gis' 'public',
```

Comment:
Comma Missing, This causes concatenation which makes this string "django.contrib.gispublic" instead of "django.contrib.gis", "public"